icewm-confirm_logout-20160608.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": [],
+  "area": [
+    {
+      "xpos": 360,
+      "ypos": 320,
+      "width": 304,
+      "height": 126,
+      "type": "match"
+    }
+  ],
+  "tags": [
+    "confirm_logout"

```

Comment:
I guess it should be a bit more specific, e.g. icewm-confirm_logout, like the filename.